<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>509</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/resources/cursorBlackWhite.png</normaloff>:/resources/cursorBlackWhite.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0" rowminimumheight="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Saved_Game"/>
   <addaction name="actionSave_Game"/>
   <addaction name="actionSettings"/>
   <addaction name="actionPlay_on_Bluetooth"/>
   <addaction name="actionPlay_on_Network"/>
   <addaction name="actionAdjust_for_Small_Display"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Log viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="logView">
       <property name="font">
        <font>
         <family>Liberation Mono</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/preferences-other.svg</normaloff>:/resources/oxygen-icons/preferences-other.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave_Game">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/document-save.svg</normaloff>:/resources/oxygen-icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save Game</string>
   </property>
  </action>
  <action name="actionOpen_Saved_Game">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/open-saved-game--folder-open.svg</normaloff>:/resources/oxygen-icons/open-saved-game--folder-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open Saved Game</string>
   </property>
  </action>
  <action name="actionPlay_on_Bluetooth">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/preferences-system-bluetooth.svg</normaloff>:/resources/oxygen-icons/preferences-system-bluetooth.svg</iconset>
   </property>
   <property name="text">
    <string>Play on Bluetooth</string>
   </property>
   <property name="toolTip">
    <string>Play a Game on Bluetooth</string>
   </property>
  </action>
  <action name="actionPlay_on_Network">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/playerNetwork--network-wired.svg</normaloff>:/resources/oxygen-icons/playerNetwork--network-wired.svg</iconset>
   </property>
   <property name="text">
    <string>Play on Network</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/system-help.svg</normaloff>:/resources/oxygen-icons/system-help.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAdjust_for_Small_Display">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/phone.svg</normaloff>:/resources/oxygen-icons/phone.svg</iconset>
   </property>
   <property name="text">
    <string>Adjust for Small Display</string>
   </property>
   <property name="toolTip">
    <string>Optimise the game for a small resolution</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/dialog-information.svg</normaloff>:/resources/oxygen-icons/dialog-information.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDebug_BT">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/resources/oxygen-icons/kbugbuster.svg</normaloff>:/resources/oxygen-icons/kbugbuster.svg</iconset>
   </property>
   <property name="text">
    <string>Debug BT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
